pytest/tests/test_signature_request.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+"""
+Starts 2 near validators and 2 mpc nodes.
+Deploys mpc contract and sends a signature request.
+Verifies that the mpc nodes index the signature request.
```

Comment:
This test covers more than just indexing now; we could update this comment as well as the function name below.